src/dsp_tools/models/ontology.py

```diff
@@ -210,23 +210,19 @@ def fromJsonObj(cls, con: Connection, json_obj: Any) -> "Ontology":
         #
         # First let's get the ID (IRI) of the ontology
         #
-        id = json_obj.get("@id")
-        if id is None:
-            raise BaseError("Ontology id is missing")
+        iri = json_obj.get("@id")
+        if iri is None:
+            raise BaseError("Ontology iri is missing")
 
         #
         # evaluate the JSON-LD context to get the proper prefixes
         #
         context = Context(json_obj.get("@context"))
-        onto_name = id.split("/")[-2]
-        context.add_context(onto_name, id + "#")
-        rdf = context.prefix_from_iri("http://www.w3.org/1999/02/22-rdf-syntax-ns#")
+        onto_name = iri.split("/")[-2]
+        context.add_context(onto_name, iri + "#")
         rdfs = context.prefix_from_iri("http://www.w3.org/2000/01/rdf-schema#")
-        owl = context.prefix_from_iri("http://www.w3.org/2002/07/owl#")
```

Comment:
unused variables
(in fact, all lines that I deleted without comment were unused)

Reply:
I'm not 100% sure about this... but couldn't it possibly be nice to have those in the context? like this you can use `owl:SomethingFromOwl` in your requests/responses, without ever having to worry about wether or not `owl` is defined in your context.  
Not sure about the use case, though.

Also, this kind of thing should live in a separate json-ld-context util file for reuseability

Reply:
AFAIU, the deleted LOC doesn't have side effects, it just retrieves a value. Some of the values that are retrieved are used later, but some (like the deleted LOC), weren't used. (`owl` was an unused variable).

So I don't think that we lose something if I delete them. Or did I miss something here?

Reply:
no, probably you're right (and if not, the code is in version control, so it's brought back easily enough)